core/internal/cltest/job_factories.go

```diff
@@ -90,15 +94,31 @@ func CompareOCRJobSpecs(t *testing.T, expected, actual job.Job) {
 	require.Equal(t, expected.OffchainreportingOracleSpec.ContractConfigConfirmations, actual.OffchainreportingOracleSpec.ContractConfigConfirmations)
 }
 
-func MustInsertWebhookSpec(t *testing.T, db *gorm.DB) (job.Job, job.WebhookSpec) {
+func MustInsertWebhookSpec(t *testing.T, db *sqlx.DB) (job.Job, job.WebhookSpec) {
+	jobORM, pipelineORM := getORMs(t, db)
 	webhookSpec := job.WebhookSpec{}
-	err := db.Create(&webhookSpec).Error
+	webhookSpecID, err := jobORM.InsertWebhookSpec(&webhookSpec)
```

Comment:
It might be preferable to rewrite `InsertWebhookSpec` so that instead of returning the id it uses `RETURNING *` to scan into the passed pointer.